operations/observability/mixins/IDE/rules/vscode.yaml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2022 Gitpod GmbH. All rights reserved.
+# Licensed under the GNU Affero General Public License (AGPL).
+# See License-AGPL.txt in the project root for license information.
+
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  labels:
+    prometheus: k8s
+    role: alert-rules
+  name: openvsx-proxy-monitoring-rules
+  namespace: monitoring-satellite
+spec:
+  groups:
+  - name: vscode
+    rules:
+    - alert: VSCodeExtensionInstallFailuresRatioTooHigh
+      labels:
+        severity: critical
+      for: 20m
+      annotations:
+        runbook_url: https://github.com/gitpod-io/runbooks/blob/main/runbooks/VSCodeExtensionInstallFailuresRatioTooHigh.md
```

Comment:
the link does not seem to work

Reply:
It will work after this PR https://github.com/gitpod-io/runbooks/pull/405